assets/sass/common/shared/o-forms/_o-form.scss

```diff
@@ -99,15 +99,21 @@
       box-sizing: border-box;
     }
 
+    input.password-with-toggle {
```

Comment:
This classname is added from PasswordBox component in courage. https://github.com/okta/courage/pull/475